suite-native/labeling/src/components/AccountLabel.tsx

```diff
@@ -1,28 +1,36 @@
 import { useSelector } from 'react-redux';
 
-import { WithLabelingState, selectAccountLabel } from '@suite-common/local-first-storage';
-import { AccountKey, WalletDescriptor } from '@suite-common/wallet-types';
+import {
+    WithLabelingState,
+    selectAccountLabel as selectAccountLabelLocalFirst,
+} from '@suite-common/local-first-storage';
+import { AccountsRootState, selectAccountLabel } from '@suite-common/wallet-core';
+import type { Account } from '@suite-common/wallet-types';
+import { parseDeviceStaticSessionId } from '@suite-common/wallet-utils';
 import { Text } from '@suite-native/atoms';
 
 import { useIsLabelingEnabled } from './useIsLabelingEnabled';
 
-type AccountLabelProps = {
-    walletDescriptor: WalletDescriptor;
-    accountKey: AccountKey;
-    /** @deprecated this shall be removed once Evolu labeling is rolled out and account.accountLabel data migrated */
-    fallbackLabel: string | undefined;
-};
-
-export const AccountLabel = ({
-    accountKey,
-    walletDescriptor,
-    fallbackLabel,
-}: AccountLabelProps) => {
+export const useAccountLabel = ({ account }: { account: Account | null }) => {
```

Comment:
I know it is sa bit more, but I basically jut passed there the account entity. Reason being, that you need it either way anywhere you'd use the AccountLabel.

Sure, it is not greatest practice to pass things you don't need, but it saves a lot of selectors.